github/MainClass.py

```diff
@@ -808,6 +808,7 @@ def get_app(self, slug=github.GithubObject.NotSet):
         if slug is github.GithubObject.NotSet:
```

Comment:
I'd make `slug` mandatory and remove the `=github.GithubObject.NotSet` and `if slug is github.GithubObject.NotSet:`. This method should only serve the current `else` branch.

Reply:
ignore this, see below...